### DISCUSSION TITLE
database connection example

### DISCUSSION THREAD

**Discussion by holeefuk** [GoogleChrome/lighthouse-ci#600] [Q&A]
Hello,

Are there any examples of how to connect to the sqlite database when starting the ci server with the example command provided ?

npx lhci server --storage.storageMethod=sql --storage.sqlDialect=sqlite --storage.sqlDatabasePath=./db.sql

Thank you in advance
R

**Comment by patrickhulce** [comment]
Are you asking how to connect to the sqlite database manually without Lighthouse CI involved if that's the LHCI command you're using to start the server?

If so, that'd just be `sqlite ./db.sql`.
